packages/core/src/utils.ts

```diff
@@ -68,7 +68,7 @@ export function getAltTranslationFileGlob(config: UserConfig) {
     altLanguages.length === 1 ? altLanguages[0] : `{${altLanguages.join(',')}}`;
 
   const { translationsDirectorySuffix = defaultTranslationDirSuffix } = config;
-  const result = `**/*${translationsDirectorySuffix}/${langMatch}.translations.json`;
+  const result = `**/?(*)${translationsDirectorySuffix}/${langMatch}.translations.json`;
```

Comment:
`fast-glob` has some weird glob rules that no other library has. `tinyglobby` wasn't finding alternative translations until I made this change.